config.ts

```diff
@@ -71,21 +72,22 @@ const contractAddressesSchema = yup.object().shape({
 const contractConfigSchema = yup.object().shape({
   JUNIOR_OPERATOR: yup
     .mixed<'ALLOWANCE_OPERATOR'>()
-    .required('contractConfigSchema.JUNIOR_OPERATOR is required')
+    // .required('contractConfigSchema.JUNIOR_OPERATOR is required')
     .oneOf(['ALLOWANCE_OPERATOR']),
   SENIOR_OPERATOR: yup
     .mixed<'PROPORTIONAL_OPERATOR' | 'ALLOWANCE_OPERATOR'>()
-    .required('contractConfigSchema.SENIOR_OPERATOR is required')
+    // .required('contractConfigSchema.SENIOR_OPERATOR is required')
     .oneOf(['PROPORTIONAL_OPERATOR', 'ALLOWANCE_OPERATOR']),
 })
 
 // TODO: text/logo/etc should be required and description/investHtml removed,
 // once we migrate all pool configs to the new format
 const poolSchema = yup.object().shape({
-  addresses: contractAddressesSchema.required('poolSchema.addresses is required'),
+  addresses: contractAddressesSchema,
```

Comment:
Same here?

Reply:
Since you've split up the upcoming pool and open/closed pool schemas, I think these requires can be added again.